pkg/security/podsecuritypolicy/util/util.go

```diff
@@ -182,8 +183,9 @@ func PSPAllowsHostVolumePath(psp *extensions.PodSecurityPolicy, hostPath string)
 		return true
 	}
 
+	path := filepath.Clean(hostPath)
```

Comment:
I'm actually more surprised that hostpath mounts paths with backsteps. @kubernetes/sig-storage-bugs